src/features/cartographie/components/lieux-mediation-numerique-details/lieux-mediation-numerique-details.component.ts

```diff
@@ -19,4 +19,8 @@ export class LieuxMediationNumeriqueDetailsComponent {
   @Output() public closeDetails: EventEmitter<LieuMediationNumeriqueDetailsPresentation> =
     new EventEmitter<LieuMediationNumeriqueDetailsPresentation>();
   @Output() public showLabel: EventEmitter<LabelNational> = new EventEmitter<LabelNational>();
+
+  public lieuIsFranceServices(): boolean {
+    return this.lieuMediationNumerique.labels_nationaux?.includes('France Services' as LabelNational) ?? false;
```

Comment:
Je pense que tu peux faire un truc comme ça pour éviter le `as` :
```suggestion
    return this.lieuMediationNumerique.labels_nationaux?.includes(LabelNational.FranceServices) ?? false;
```